SCRIPTS.md

```diff
@@ -93,3 +93,4 @@
 | 89\.     | Voice-Assistant  | Voice-Assistant Bot | [Take Me](https://github.com/avinashkranjan/Amazing-Python-Scripts/tree/master/Voice-Assistant) |  [Avinash .K. Rajan](https://github.com/avinashkranjan) |
 | 90\.     | Website-Status-Checker  | This script will check the status of the web address which you will input | [Take Me](https://github.com/avinashkranjan/Amazing-Python-Scripts/tree/master/Website-Status-Checker) |  [Avinash .K. Rajan](https://github.com/avinashkranjan) |
 | 91\.     | Movie-Genre-Prediction-Chatbot       | An chatbot will predict the genre of the movie based on the input text. | [Take Me](https://github.com/avinashkranjan/Amazing-Python-Scripts/tree/master/Movie-Genre-Prediction-Chatbot) | [Hritik Jaiswal](https://github.com/hritik5102) |
+| 92\.     | Tarot-Reader       | A script that draws tarot cards and gives fortunes | [Take Me](https://github.com/avinashkranjan/Amazing-Python-Scripts/tree/tarot-reader/Tarot%20Reader) | [Akshaya Kulasekaran](https://github.com/Akshu-on-github) |
```

Comment:
Script adding process will soon be automated therefore you don't need to add it manually. Please remove your changes from this file

Reply:
I've removed the same